packages/jaeger-ui-components/src/TraceTimelineViewer/VirtualizedTraceView.tsx

```diff
@@ -495,11 +509,13 @@ export class UnthemedVirtualizedTraceView extends React.Component<VirtualizedTra
     );
   }
 
+  scrollToTopTraceView = () => this.topTraceViewRef.current?.scrollIntoView();
```

Comment:
I think we should scroll to the top of the page, or add some top offset to the scroll. The current behavior makes me feel like I'm still deep in the trace. Wdyt?